src/components/scenegraph/SceneGraph.js

```diff
@@ -227,6 +237,14 @@ export default class SceneGraph extends React.Component {
   }
 
   render () {
+    // to hide the SceneGraph we have to hide its parent too (#left-sidebar)
+    var leftsidebar = document.getElementById('left-sidebar');
```

Comment:
I've deleted the extra div for the `left-sidebar` and now we've just the scenegraph with all the properties (and the ID too in fact) of the previous `left-sidebar` so you won't need this :)